src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -994,4 +1063,25 @@ private Response isNameUnique(Policy policy) {
 
         return null;
     }
+
+    private Response reloadEngineTriggers(String accountId, Set<UUID> policyIds, Supplier<ResponseBuilder> successResponseSupplier) {
+        try {
+            lightweightEngine.reloadTriggers(accountId, policyIds);
+        } catch (Throwable t) {
+            log.error("Failed to reload engine triggers", t);
+            try {
+                // Any change previously done to the DB data during the current transaction will be rolled back.
+                transactionManager.setRollbackOnly();
```

Comment:
How does this work? 
The engine takes the ids, but are the values available to be read by the engine if they are not commited? 
Also - what happens if the engine is down - do we really need it for the policy CRUD ?

Reply:
Good question! I haven't started the local tests yet but thanks to your comment I can already tell that there will be an issue here 😄 Not a big one though. I'll probably move the transactions boundaries as everything needs to be comitted in `ui-backend` before we call `engine`.

Reply:
This may actually encourage me to do something I had in mind and didn't implement yet. I may have been wrong writing on Slack that there wouldn't be any major code change. 😄